templates/frontOffice/default/account.html

```diff
@@ -60,7 +60,7 @@ <h4 class="panel-title">
                                     {if $CELLPHONE != ""}
                                         <span class="mobile">{$CELLPHONE}</span>
                                     {/if}
-                                    <span class="email">{mailto address="{$EMAIL}" encode="hex"}</span>
+                                    <span class="email">{mailto address=$EMAIL encode="hex"}</span>
```

Comment:
The lord of XSS has spoken. This can't be merged.
![hallelujah](https://cloud.githubusercontent.com/assets/6106094/6575641/eb1f0260-c730-11e4-9f2a-e6b6d1c4b7e3.gif)